src/script/base.ts

```
@@ -80,6 +95,138 @@ export class VtxoScript {
+        CSVMultisigTapscript.Type | ConditionCSVMultisigTapscript.Type
+    > {
+        const paths: Array<
+            CSVMultisigTapscript.Type | ConditionCSVMultisigTapscript.Type
+        > = [];
+        for (const leaf of this.leaves) {
+            try {
+                const tapscript = CSVMultisigTapscript.decode(
+                    scriptFromTapLeafScript(leaf)
+                );
+                paths.push(tapscript);
+                continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)